### PR TITLE
FIX_649: Force command line if dep cannot be solved

### PR DESCRIPTION
**Pull request by raxyu**

### CODE REVIEW

File: cloudbackup_updater/pkgup/_yum/updater.py [hunk 1]
```diff
@@ -67,7 +69,7 @@ def __installed(self):
     def __run_cmdline(self, command, version=None):
             LOG.debug('Running command line')
 
-            p = subprocess.Popen(['yum', command, "--disablerepo='*'",
+            p = subprocess.Popen(['yum', command, "localinstall", "--disablerepo='*'",
```
Comment (jc7998) [inline cloudbackup_updater/pkgup/_yum/updater.py]: Not sure about localinstall.
localinstall seems to be reserved for installing .rpm packages available in the local directory.

File: setup.py [hunk 2]
```diff
@@ -8,10 +8,10 @@ def read(fname):
 
 setup(
     name='cloudbackup-updater',
-    version='1.3',
+    version='1.4',
     description='Auto-updater for the Rackspace Cloud Backup agent',
-    author='Zhihao Yuan',
-    author_email='zhihao.yuan@rackspace.com',
+    author='Cloud Backup Dev',
+    author_email='',
```
Comment (brett-johnson) [inline setup.py]: Should we use CloudBackup@rackspace.com here?

Comment (raxyu) [inline setup.py]: Fixed.  Thanks!

**Comment by BenjamenMeyer** [comment]
:+1:

**Comment by lichray** [comment]
I didn't noticed that the purpose of this change is to skip rsync dep checking.  I manually added the librsync dependence to rpm packaging script (install.txt something in agent repo); remove it from there please.

**Comment by raxyu** [comment]
Hello Zhihao,  

If you are talking about agent/solution/package/linux/install_text.py, then, yes, that's what I did.  But thank you very much for your reminding!  

Best, 

-Xuan

**Comment by lichray** [comment]
Yes, then what's the use of this change?

**Comment by brett-johnson** [comment]
Hard to let go, isn't it, Zhihao? :)

**Comment by raxyu** [comment]
The updater has trouble to install new dependence in yum, such as librsync.  We hope this one will fix it.

**Comment by lichray** [comment]
If you don't want librsync to be a dependence, then remove the dependence from `agent/solution/package/linux/install_text.py`, why change installer?  If after the removal, librsync is still a dependence in the rpm package, that means my changes to 1.33 is not fully merged to master.

**Comment by BenjamenMeyer** [comment]
@lichray the issue is that cloudbackup-updater is having problems installing extra depedendencies, like librsync so we have to fix it to do so. We're temporarily removing the dependency from cloudbackup-agent so that the existing cloudbackup-updaters out there will install the new cloudbackup-updater with the fix so that the next agent push can restore the dependency again.

**Comment by lichray** [comment]
OK.  I didn't see any, but Jorge may found issues here.

**Pull request merged**